lib/utils.pm

```diff
@@ -1406,7 +1406,11 @@ sub reconnect_mgmt_console {
     elsif (check_var('ARCH', 'x86_64')) {
         if (check_var('BACKEND', 'ipmi')) {
             select_console 'sol', await_console => 0;
-            assert_screen [qw(qa-net-selection prague-pxe-menu)], 300;
+            if (!get_var('IPXE_CONSOLE')) {
+                assert_screen [qw(qa-net-selection prague-pxe-menu)], 300;
+            } else {
+                check_screen 'grub2', 300;
+            }
```

Comment:
Please add some comment  on issue with sol on sonic/tails.

Reply:
`assert_screen [qw(qa-net-selection prague-pxe-menu)], 300 unless if get_var('IPXE_CONSOLE')`;
should work too